api/wasm/indigo-ketcher/indigo-ketcher.cpp

```diff
@@ -349,9 +349,10 @@ namespace indigo
     class VectorStringBracketed
     {
         const std::vector<std::string>& vec;
+        bool both{false};
```

Comment:
Consider to move from single vector + both to pair of vectors.
Since we need bracketed object only for reactions, it supposes two parts. Let's model "two parts" not outside this class by vector[2], but inside this specific class. In this case, the caller can use: 

instead `molWeights[0]` -> `molWeights.products`
instead `molWeights[1]` -> `molWeights.reactants`

Reply:
done